src/tools/run-make-support/src/run.rs

```diff
@@ -43,7 +48,20 @@ pub fn run(name: &str) -> Output {
     let caller_location = std::panic::Location::caller();
     let caller_line_number = caller_location.line();
 
-    let (cmd, output) = run_common(name);
+    let (cmd, output) = run_common(name, None);
+    if !output.status.success() {
+        handle_failed_output(&cmd, output, caller_line_number);
+    }
+    output
+}
+
+/// Run a built binary with one or more argument(s) and make sure it succeeds.
+#[track_caller]
+pub fn run_with_args(name: &str, args: &[&str]) -> Output {
```

Comment:
Thoughts for myself: do we want to have a `run_with_args` versus `run`? I guess it might be nicer than having to write `run(..., &[])` for the common no-arg cases?

Reply:
`run` with no args is extremely common (many of the tests I ported use it) and I have seen very few run calls with arguments, this test being one rare exception. I think separate functions is better.